### ISSUE TITLE
[Illustrated Message] a11y review

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#1276]
### Feedback

- How does someone create a heading at a level other than H2. What if this is not appropriate?
- Recommendations need to be added to the examples for how to mark up the SVGs to work correctly. Decorative and non decorative examples should be added

### Path forward

1. Review attribute + slot relationship for wider customization options of headline and possibly description content
2. Discuss usage of decorative and non-decorative SVGs with Spectrum for recommendations on what content to demonstrate in this area.

cc @jnurthen

**Comment by Westbrook** [comment]
@jnurthen thoughts on closing this in favor of the more recent review cycle?

**Comment by Westbrook** [comment]
Closing in preference on newer review.
